platforms/software/testing-base/src/integTest/groovy/org/gradle/testing/CustomTestTaskHTMLTestReportMetadataTest.groovy

```diff
@@ -339,4 +345,9 @@ final class CustomTestTaskHTMLTestReportMetadataTest extends AbstractIntegration
             tasks.register("${name}", ${name}CustomTestTask)
         """
     }
+
+    private String normalizePath(String relativePath) {
+        boolean isWindows = System.getProperty("os.name").toLowerCase().contains("windows")
```

Comment:
Can we use our normal OS detection logic here, with `org.gradle.internal.os.OperatingSystem.current()`? This already has a potential bug in it and I don't see any reason to not use it.

Reply:
Sounds good, I'll update.